system-parachains/constants/src/lib.rs

```diff
@@ -59,3 +59,36 @@ pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
 
 /// Treasury pallet id of the local chain, used to convert into AccountId
 pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"py/trsry");
+
+/// Parameters supporting async backing functionality.
+///
+/// Once all system chains have migrated to the new async backing mechanism, the parameters
+/// in this namespace will replace those currently defined in `super::*`.
+pub mod async_backing {
+	use frame_support::weights::{constants::WEIGHT_REF_TIME_PER_SECOND, Weight};
+	pub use parachains_common::BlockNumber;
+	use sp_runtime::Perbill;
+
+	/// The average expected block time that we are targeting.
+	pub const MILLISECS_PER_BLOCK: u64 = 6_000;
+	pub const SLOT_DURATION: u64 = 12_000;
+
+	// Time is measured by number of blocks.
+	pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
+	pub const HOURS: BlockNumber = MINUTES * 60;
+	pub const DAYS: BlockNumber = HOURS * 24;
+
+	/// We assume that ~5% of the block weight is consumed by `on_initialize` handlers. This is
+	/// used to limit the maximal weight of a single extrinsic.
+	pub const AVERAGE_ON_INITIALIZE_RATIO: Perbill = Perbill::from_percent(5);
```

Comment:
Is this not normally higher? I know that we have more ref time now with async backing, but not more PoV size, or?  
It should still be 5MiB?

Reply:
Right, it use to be 5%. Still will be higher in absolute numbers.